src/smart-components/group/add-group/summary-content.js

```diff
@@ -35,41 +35,41 @@ const SummaryContent = (formData) => {
             <StackItem>
```

Comment:
instead of adding the `pf-u-mb-md` to each of the `Text` components, can we just wrap all of this in another `<Stack>` and then add a gutter?